networking/igmp/conformance/common/include.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# variables to control some default action
+NM_CTL=${NM_CTL:-"no"}
+FIREWALL=${FIREWALL:-"no"}
+AVC_CHECK=${AVC_CHECK:-"yes"}
+if [ ! "$JOBID" ]; then
+    RED='\E[1;31m'
+    GRN='\E[1;32m'
+    YEL='\E[1;33m'
+    RES='\E[0m'
+fi
+
+new_outputfile()
+{
+    mktemp /mnt/testarea/tmp.XXXXXX
+}
+
+
+test_pass()
+{
+    #SCORE=${2:-$PASS}
+    echo -e "\n:: [  PASS  ] :: Test '"$1"'" | tee -a $OUTPUTFILE
+    # we don't care how many test passed
+    if [ $JOBID ]; then
+       report_result "${TEST}/$1" "PASS"
+    else
+        echo -e "\n::::::::::::::::"
+        echo -e ":: [  ${GRN}PASS${RES}  ] :: Test '"${TEST}/$1"'"
+        echo -e "::::::::::::::::\n"
+    fi
+}
+
+test_fail()
+{
+    SCORE=${2:-$FAIL}
+    echo -e ":: [  FAIL  ] :: Test '"$1"'" | tee -a $OUTPUTFILE
+    # we only care how many test failed
+    if [ $JOBID ]; then
+        report_result "${TEST}/$1" "FAIL" "$SCORE"
+    else
+        echo -e "\n:::::::::::::::::"
+        echo -e ":: [  ${RED}FAIL${RES}  ] :: Test '"${TEST}/$1"' FAIL $SCORE"
+        echo -e ":::::::::::::::::\n"
+    fi
+}
+
+test_warn()
+{
+    echo -e "\n:: [  WARN  ] :: Test '"$1"'" | tee -a $OUTPUTFILE
+    if [ $JOBID ]; then
+        report_result "${TEST}/$1" "WARN"
+    else
+        echo -e "\n:::::::::::::::::"
+        echo -e ":: [  ${YEL}WARN${RES}  ] :: Test '"${TEST}/$1"'"
+        echo -e ":::::::::::::::::\n"
+    fi
+}
+
+# We only care the main distro
+GetDistroRelease()
```

Comment:
Will this work fine on Fedora too? It still returns the distro version and that's only being compared with 7 in the test so I'm guessing it should be fine. Just putting this note out here to keep it in mind in case we can actually run upstream kernels on Fedora and not RHEL in the future, not something that needs to be changed in the test right now :)

Reply:
I think it's ok right now since these are workarounds for rhel6